filebeat/tests/system/test_processors.py

```diff
@@ -115,3 +115,56 @@ def test_condition(self):
         assert "beat.name" in output
         assert "message" in output
         assert "test" in output["message"]
+
+    def test_dissect_good_tokenizer(self):
+        """
+        Check dissect with a good tokenizer
+        """
+        self.render_config_template(
+            path=os.path.abspath(self.working_dir) + "/test.log",
+            processors=[{
+                "dissect": {
+                    "tokenizer": "\"%{key} world\"",
+                    "field": "message",
+                    "target_prefix": "extracted"
```

Comment:
If you leave this one out, you can test the defaults.

Reply:
I have added something specific to test all the defaults instead, I prefer clearer intend.